File: core/src/backendServer/models/BouncerPickup.js
const Pickup = require('./Pickup');
const constants = require('../settings/constants');
const ProjectileType = require('./ProjectileType');

class BouncerPickup extends Pickup {
    constructor(x, y, id, ammunition = constants.BOUNCER_BULLETS_IN_CHAMBER) {
        super(x, y, constants.BOUNCER_SPRITE_WIDTH, constants.BOUNCER_SPRITE_HEIGHT, ProjectileType.BOUNCER,
            id, ammunition)
    }
}

module.exports = BouncerPickup;

File: core/src/com/mygdx/game/model/pickup/BouncerPickup.kt
package com.mygdx.game.model.pickup

import com.badlogic.gdx.graphics.Texture
import com.mygdx.game.settings.BOUNCER_SPRITE_HEIGHT
import com.mygdx.game.settings.BOUNCER_SPRITE_WIDTH

class BouncerPickup(
        x: Float = 0f,
        y: Float = 0f,
        texture: Texture) : Pickup(
        x,
        y,
        BOUNCER_SPRITE_WIDTH,
        BOUNCER_SPRITE_HEIGHT,
        texture
)